ccd-definition/CaseEventToComplexTypes/AddOrAmendClaimDocuments.json

```diff
@@ -52,9 +52,18 @@
     "CaseFieldID": "servedDocumentFiles",
     "ListElementCode": "medicalReport.document",
     "FieldDisplayOrder": 6,
+    "DisplayContext": "OPTIONAL"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Upload",
+    "CaseEventID": "CREATE_CLAIM",
+    "CaseFieldID": "servedDocumentFiles",
+    "ListElementCode": "medicalReport",
     "DisplayContext": "OPTIONAL",
-    "Min": 1,
-    "FieldShowCondition": "claimType=\"CLINICAL_NEGLIGENCE\" OR claimType=\"PERSONAL_INJURY\""
+    "FieldDisplayOrder": 6,
```

Comment:
is it 6 or 7 ?

Reply:
it is the same as ```medicalReport.document``` so should be 6 in this case